charts/openstack-cluster/values.yaml

```diff
@@ -378,8 +378,8 @@ nodeGroupDefaults:
     spec:
       # By default, unhealthy worker nodes are always remediated
       maxUnhealthy: 100%
```

Comment:
# (Optional) maxUnhealthy prevents further remediation if the cluster is already partially unhealthy
  maxUnhealthy: 40%

Maybe 100% is the wrong call here. I think its actually a limit around remediation.

Reply:
I think we might want to set this number to 1, or something like that, so it limits the number of unhealth nodes we rebuild at any one time?
https://cluster-api.sigs.k8s.io/tasks/automated-machine-management/healthchecking#max-unhealthy